Sources/TensorFlow/Loss.swift

```diff
@@ -24,6 +24,23 @@ public func meanSquaredError<Scalar: TensorFlowFloatingPoint>(
     return (expected - predicted).squared().mean()
 }
 
+/// Returns the mean squared logarithmic error between predictions and expectations.
+///
+/// - Note: Negative Tensor entries will be clamped at 0 to avoid undefined
```

Comment:
In documentation comments, we usually wrap numeric values in backquotes.

```suggestion
/// - Note: Negative Tensor entries will be clamped at `0` to avoid undefined
```